app/models/series.rb

```diff
@@ -104,4 +108,23 @@ def schedule_for_episode_number(episode_number)
 
     schedule_index
   end
+
+  def destroy
+    if v4?
+      really_destroy!
```

Comment:
I guess that is what I mean - why not make them all set deleted_at, why fully delete v4?